cvat/apps/engine/static/engine/js/annotationUI.js

```diff
@@ -506,9 +506,13 @@ function buildImageSizeGetter(isExternal, sizeByFrame, imageMetaData) {
 
 function buildImageUrlGetter(isExternal, urlByFrame, taskId) {
     if (isExternal) {
-        return (frame, gamma=1) => {
-            const url = urlByFrame[frame];
-            return `${url}?gc=${gamma}`;
+        return (frame, params) => {
+            const frameUrl = urlByFrame[frame];
+            paramsUrl = `gc=${params.gc}`;
+            if (params.white_balance) {
+                paramsUrl = `white_balance,${paramsUrl}`;
```

Comment:
this doesn't look like the format I made the records server give me, which was
..../"cam0/1636642997436417618.jpg?ie=white_balance,gc:1"

Reply:
Great catch!